docs/example_static_html.md

```diff
@@ -3,50 +3,176 @@ title: "Example: Plain Old Static HTML"
+serves static HTML.
 
-Oneup is a simple app that uses busybox to serve HTML files.
+We'll use Tilt to:
 
-First, check out the Tilt repo.
+- Run the server on Kubernetes
+- Measure the time from a code change to a new process
+- Optimize that time for fast feedback
 
+Obviously, this is a silly example. But it can be a useful example to confirm that Tilt is working
+as expected in your environment.
+
+All the code is in this repo:
+
+[tilt-example-html](https://github.com/windmilleng/tilt-example-html)
+
+To skip straight to the fully optimized setup, go to this subdirectory:
+
+[Recommended Tiltfile](https://github.com/windmilleng/tilt-example-html/blob/master/2-recommended/Tiltfile)
+
+## Step 0: The Simplest Deployment
+
+Our server is a two-line shell script:
+
+```shell
+echo "Serving files on port 8000"
+busybox httpd -f -p 8000
 ```
-git clone https://github.com/windmilleng/tilt
-cd tilt/integration/oneup
+
+To start this server on Kubernetes, we need 3 configs:
+
+1) A [Dockerfile](https://github.com/windmilleng/tilt-example-html/blob/master/0-base/Dockerfile) that builds the image
+
+2) A [Kubernetes deployment](https://github.com/windmilleng/tilt-example-html/blob/master/0-base/kubernetes.yaml) that runs the image
+
+3) And finally, a Tiltfile that ties them together
+
+```python
+docker_build('example-html-image', '.')
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-html', port_forwards=8000)
 ```
 
-We'll be looking at the [oneup](https://github.com/windmilleng/tilt/tree/master/integration/oneup) project
-in `integration/oneup`.
+The first line tells Tilt to build an image with the name `example-html-image`
+in the directory `.` (the current directory).
+
+The second line tells Tilt to load the Kubernetes
+[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment)
+yaml. The image name in the `docker_build` call must match the container `image`
+reference in the `example-html` Deployment.
+
+The last line configures port-forwarding so that your server is
+reachable at http://localhost:8000/. The resource name in the `k8s_resource` call
+must match the Deployment's `metadata.name` in `kubernetes.yaml`.
 
-In the oneup directory, run
+Try it! Run:
 
 ```
+git clone https://github.com/windmilleng/tilt-example-html
+cd tilt-example-html/0-base
 tilt up
 ```
 
-Your terminal will turn into a status box that lets you watch your server come up. When it's ready,
-you will see the status icon turn green. The logs in the botton pane will display
-"Serving oneup on container port 8000."
+Your terminal will turn into a status box that lets you watch your server come
+up. Tilt will open a browser view of the same status.
+
+When it's ready, you will see the status icon turn green. The logs in the
+botton pane will display "Serving files on port 8000."
+
+[![Step 0](assets/docimg/example-static-html-image-1.png)](https://cloud.tilt.dev/snapshot/AejkyuULr2AjWu50Eck=){:.is-image}
```

Comment:
consider adding a caption (with or without descriptive information like "the server is up!") that includes "click the image to see an interactive snapshot"--cool change to show off snapshot functionality, and users might not think to click otherwise.

Reply:
done